package NGroundProto;


message TLabel {
    optional uint32 LabelId = 1;
    optional uint32 From = 2;
    optional uint32 To = 3;
}


message TTimeSeriesInfo {
    repeated TLabel Labels = 1;
    repeated string UniqueLabelNames = 2;
    optional double Dt = 3;
    optional uint32 DimSize = 4;
}


